js/src/setup-mc/setup-stepper/store-requirements/index.js

```diff
@@ -120,7 +136,9 @@ export default function StoreRequirements() {
 								<AppButton
 									isPrimary
 									loading={ completing }
-									disabled={ ! isReadyToComplete }
+									disabled={
+										! isReadyToComplete || ! isSaveSettings
```

Comment:
💅  Wondering if we can add `isSaveSettings` condition inside `isReadyToComplete`